[2025-02-05T06:37:51.795+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:37:51.848+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_group_example.reporting.generate_report scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2025-02-05T06:37:51.852+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_group_example.reporting.generate_report scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2025-02-05T06:37:51.852+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-05T06:37:51.857+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): reporting.generate_report> on 2024-02-07 00:00:00+00:00
[2025-02-05T06:37:51.859+0000] {standard_task_runner.py:57} INFO - Started process 1153 to run task
[2025-02-05T06:37:51.861+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'task_group_example', 'reporting.generate_report', 'scheduled__2024-02-07T00:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/task_group_dag.py', '--cfg-path', '/tmp/tmpe8lsd7l9']
[2025-02-05T06:37:51.862+0000] {standard_task_runner.py:85} INFO - Job 138: Subtask reporting.generate_report
[2025-02-05T06:37:51.868+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:37:51.881+0000] {task_command.py:415} INFO - Running <TaskInstance: task_group_example.reporting.generate_report scheduled__2024-02-07T00:00:00+00:00 [running]> on host 3849bfb82152
[2025-02-05T06:37:51.885+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:37:51.910+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_group_example' AIRFLOW_CTX_TASK_ID='reporting.generate_report' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-07T00:00:00+00:00'
[2025-02-05T06:37:51.911+0000] {logging_mixin.py:151} INFO - 执行任务: 生成报告
[2025-02-05T06:37:51.911+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-05T06:37:51.914+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=task_group_example, task_id=reporting.generate_report, execution_date=20240207T000000, start_date=20250205T063751, end_date=20250205T063751
[2025-02-05T06:37:51.959+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-05T06:37:51.970+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
